### ISSUE TITLE
About pose track code and dropping low confidence keypoints

### ISSUE THREAD

**Issue by happywu** [MVIG-SJTU/AlphaPose#9]
Hi, 
I am wondering when the tracking part code would be released?

Also, I have a question about the tracking paper, by saying "drop low-score keypoints before generating result", are you referring to keep joints that only have a confidence higher than the threshold, e.g. keep only 5 joints of a person, or referring to if the keypoints score of one person is pretty low, the entire predicted person is ruled out.  

Also, did you filter low confidence bbox first before performing tracking as the "Detect-and-Track" paper mentioned?

Many thanks.

**Comment by YuliangXiu** [comment]
Yes, you get it. 
This "drop low-score keypoints" is a keypoint-level operation, e.g. there are 14 joints for every human, I delete 7 keypoints with low-confidence(below a thresh), and then the output results will only contain 7 keypoints for this individual person.
I did filter low-confidence bbox first, but it's a little different from "Detect-and-Track", we filter few boxes by softnms with a very low thresh, this is because we need to keep enough boxes for future Pose Flow NMS process.

I will release the tracking code before the spring festival ASAP, so sorry for the latency. The deadline made me crazy  so I wrote code without tidying up.
